.github/workflows/publish.yml

```diff
@@ -23,6 +23,7 @@ jobs:
       - run: bundle install
       - run: bundle exec jekyll build
       - run: cp CNAME _site
+      - run: ./build-nuxt-site.sh
```

Comment:
This needs to be added to both workflow files: 

.github/workflows/publish.yml
.github/workflows/validate.yml

Reply:
I think you need to add:

```
      - uses: actions/setup-node@v2
        with:
          node-version: '16'
      - run: yarn install --frozen-lock-file
```